### PR TITLE
feat(ioctl): project register contract addr config

### PR DESCRIPTION
**Pull request by saitofun**
# Description

project register contract address config

Fixes #(issue)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
- [x] make test

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ioctl/cmd/ws/wsproject.go [hunk 1]
```diff
@@ -69,14 +64,7 @@ func init() {
...
 
-	wsProject.PersistentFlags().StringVarP(
-		&wsProjectRegisterContractAddress,
-		"contract-address",
-		"c",
-		"",
-		config.TranslateInLang(_flagProjectRegisterContractAddressUsages, config.UILanguage),
-	)
-	_ = wsProject.MarkFlagRequired("contract-address")
+	wsProjectRegisterContractAddress = config.ReadConfig.WsProjectRegisterContract
```
Comment (dustinxie) [inline ioctl/cmd/ws/wsproject.go:67]: why this one is special, get value from `config.ReadConfig`

File: ioctl/config/config.go [hunk 2]
```diff
@@ -69,6 +69,8 @@ type Config struct {
 	AnalyserEndpoint string            `json:"analyserEndpoint" yaml:"analyserEndpoint"`
 	// WsEndpoint w3bstream endpoint
 	WsEndpoint string `json:"wsEndpoint" yaml:"wsEndpoint"`
+	// WsProjectRegisterContract w3bstream project register contract address
+	WsProjectRegisterContract string `json:"wsProjectRegisterContract" yaml:"wsProjectRegisterContract"`
```
Comment (dustinxie) [inline ioctl/config/config.go:73]: nit: just make it simple `WsRegisterContract`?

**Review by dustinxie — APPROVED** [review]

File: ioctl/config/configsetget.go [hunk 3]
```diff
@@ -28,12 +28,14 @@ const (
 	_endpointPattern         = "(" + _ipPattern + "|(" + _domainPattern + ")" + "|(" + _localPattern + "))" + `(:\d{1,5})?`
 	_defaultAnalyserEndpoint = "https://iotex-analyser-api-mainnet.chainanalytics.org"
 	_defaultWsEndpoint       = "sprout-staging.w3bstream.com:9000"
+	// _defaultProjectRegisterContract  default project register contract address
+	_defaultProjectRegisterContract = "0x4F7e678B0203e0444E17512108dba4B08B39512e"
```
Comment (huangzhiran) [inline ioctl/config/configsetget.go:32]: register a new contract, for the "0x4F7e678B0203e0444E17512108dba4B08B39512e" contains much localhost uri

File: ioctl/newcmd/config/config.go [hunk 4]
```diff
@@ -35,12 +35,14 @@ const (
 	_defaultAnalyserEndpoint = "https://iotex-analyser-api-mainnet.chainanalytics.org"
 	_defaultWsEndpoint       = "sprout-staging.w3bstream.com:9000"
 	_defaultConfigFileName   = "config.default"
+	// _defaultWsProjectRegisterContract default w3bstream project register contract address
+	_defaultWsProjectRegisterContract = "0x4F7e678B0203e0444E17512108dba4B08B39512e"
```
Comment (huangzhiran) [inline ioctl/newcmd/config/config.go:39]: same above

**Review by huangzhiran — APPROVED** [review]

**Pull request merged**